src/Behat/MinkExtension/ServiceContainer/Driver/ZombieFactory.php

```diff
@@ -44,6 +44,10 @@ public function configure(ArrayNodeDefinition $builder)
                 ->scalarNode('server_path')->defaultNull()->end()
                 ->scalarNode('threshold')->defaultValue(2000000)->end()
                 ->scalarNode('node_modules_path')->defaultValue('')->end()
+                ->arrayNode('options')
```

Comment:
@stof has @berliner changed what you've requested. If so, then this PR can be merged.